README.md

```diff
@@ -1,8 +1,11 @@
+# Sorcery: Magical Authentication
+
+[![Gem Version](https://badge.fury.io/rb/sorcery.svg)](https://rubygems.org/gems/sorcery)
+[![Gem Downloads](https://img.shields.io/gem/dt/sorcery.svg)](https://rubygems.org/gems/sorcery)
 [![Build Status](https://travis-ci.org/NoamB/sorcery.svg?branch=master)](https://travis-ci.org/NoamB/sorcery)
```

Comment:
Change to `Sorcery/sorcery`? Also applies to lines 6 and 7. (Possibly more elsewhere too.)